src/odemis/acq/stitching/_registrar.py

```diff
@@ -14,18 +14,442 @@
 
 You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
 '''
-from __future__ import division
-
 
 # This is a series of classes which use different methods to compute the "best
 # location" of an image set based on their metadata and content. IOW, it does
 # "image registration".
 
-# TODO: define a generic API: can take a set of images, with some being explicitly
-# linked/locked in position, and allow to add new images on the set (to run it
-# in live mode).
 
-# TODO: simple version which returns the data as-is
+from __future__ import division
+from odemis.acq.drift import CalculateDrift
```

Comment:
We really need to rename {{{CalculateDrift}}} to something more appropriate. You can do it in a separate pull request.